social/api/tests.py

```diff
@@ -0,0 +1,56 @@
+# Django
+from django.test import TestCase
+from django.contrib.auth.models import User
+# Django REST
+from rest_framework.test import APIClient, APITestCase
+# App
+from author.models import Author
+from post.models import Post
+from api.posts import *
+# Python
+import json
+
+# Create your tests here.
+class PostApiTestCase(TestCase):
+    def setUp(self):
+        self.client = APIClient()
+
+        self.userObj = User.objects.create(username="TestCase__Bill__", password="test")
+        self.userObj.set_password(self.userObj.password)
+        self.userObj.save()
+        self.authorObj = Author.objects.create(id=self.userObj)
+        self.authorObj.save()
+
+
+        self.post1 = Post.objects.create(author=self.authorObj,
+                                         content="Test post 1",
+                                         privacyLevel=0)
+        self.post2 = Post.objects.create(author=self.authorObj,
+                                         content="Test post2",
+                                         privacyLevel=0)
+        self.post1.save()
+        self.post2.save()
+
+    def tearDown(self):
+        self.userObj.delete()
+        self.post1.delete()
+        self.post2.delete()
+        del self.authorObj
+        del self.userObj
+        del self.client
+        del self.post1
+        del self.post2
```

Comment:
This is just garbage cleaning the objects. Probably redundant. Django does clear it for you but I'm paranoid.